.github/workflows/ansible-test-plugins.yml

```
@@ -97,6 +101,10 @@ jobs:
           #- stable-2.15
           #- stable-2.16
           #- devel
+        clickhouse:
+          - 21.8.15.7
+          - 22.8.9.24
+          - 23.8.9.54
```

```suggestion
          - 21.8.15.7
          - 23.8.9.54
```
how about testing against the oldest supported and the newest ones?
i know now it's OK but when we add more python versions + maybe connector versions it can make a difference. What do you think?

---

It's a good option, it just doesn't take into account the features of the intermediate version, we may miss something. Can we test with three versions up to version 1.0.0, then reduce it to two?

---

sure, SGTM, we can always remove it later if needed